src/astro/databases/__init__.py

```diff
@@ -29,7 +29,6 @@ def create_database(conn_id: str) -> BaseDatabase:
     conn_type = BaseHook.get_connection(conn_id).conn_type
     module_path = CONN_TYPE_TO_MODULE_PATH[conn_type]
     module = importlib.import_module(module_path)
-    module_name = module_path.split(".")[-1]
-    class_name = module_name.title() + "Database"
+    class_name = get_class_name(module_ref=module, suffix="Database")
```

Comment:
That's great, @utkarsharma2 . Thank you very much for making them all consistent! :tada: